front/src/languages/cz.json

```diff
@@ -0,0 +1,12 @@
+{
+  "logoName": "Dětské hry",
+  "games for you": "výběr hry",
+  "Snakeee": "Had",
+  "Quiz": "Kviz",
+  "Sea battle": "Námořní bitva",
+  "Sapper": "Ženista",
+  "Mouse catcher": "Myší chytač",
+  "The best ever": "Nejlepší hra",
+  "selecttext": "Vyberte si ze seznamu",
+  "language": "jazyk"
```

Comment:
I see four different patterns for naming keys: `camelCase`, `nocaseatall`, `Upperfirst`, `names with spaces`. I'm afraid it will bring a lot of troubles in the future when the list starts growing. The other IMO good practice for improving list management is to split it into blocks, like this:
```
// en.json
{
...
"games": {
  "snakee": "Snakee",
  "quiz": "Quiz",
  ...
}
...
}

// component file
...
<div>{t(`games.${gameName}`)}</div>
...
```